clients/cypress-e2e/cypress/e2e/smoke_test.cy.ts

```diff
@@ -73,4 +74,72 @@ describe("Smoke test", () => {
       cy.get("button").contains("Test").click();
     });
   });
+
+  // TODO: add a regression test on fidesplus for legacy cookie format
```

Comment:
I'll remove this TODO- I do want to start adding some additional checks on the plus repo, but won't tie this one up